source/__tests__/integration/profile-create.test.js

```diff
@@ -14,11 +14,8 @@ describe('Sinopia Profile Editor Create Page', () => {
     });
 
     describe('header', () => {
-      it('image', async() => {
-        await expect_sel_to_exist('div.sinopia-headertext > img[src="assets/images/sinopia_profile_headertext.png"]')
-      });
```

Comment:
this image went away?  for consistency with links?

Reply:
The design changed in #146 (header spans two lines instead of one)

Reply:
More to do with consistency with the Linked Data Editor that instead of using an Image with Text, just used the Text with the correct font and size